.github/pull_request_template.md

```diff
@@ -77,3 +77,9 @@ None, automtated tests are fine.
 <!--
 ## Deployment instructions
 -->
+
```

Comment:
Please update the docs/docs/community/contributing.md to include this as it contains the PR template

Reply:
Now addressed in #470